map_test.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 var _ = Describe("Map", func() {
-	It("Should successfully load the map", func() {
-		m, err := loadMap(strings.NewReader(`bananapi/Bullseye_current|bananapi/archive/Armbian_21.08.1_Bananapi_bullseye_current_5.10.60.img.xz|Aug 26 2021|332M`))
+	It("Should successfully load the map from a CSV/Pipe separated file", func() {
+		m, err := loadMapCSV(strings.NewReader(`bananapi/Bullseye_current|bananapi/archive/Armbian_21.08.1_Bananapi_bullseye_current_5.10.60.img.xz|Aug 26 2021|332M`))
```

Comment:
Can you add a testcase for JSON, too

Reply:
It's super super basic, but I've added one. It should probably have a proper failure case/etc at some point, but it confirms that it works properly.